src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -48,17 +49,27 @@ public int Run(FileWorkItemsOptions options, IFileSystem fileSystem)
 
             SarifLog sarifLog = JsonConvert.DeserializeObject<SarifLog>(logFileContents);
 
-            FilteringStrategy filteringStrategy = FilteringStrategyFactory.CreateFilteringStrategy(options.FilteringStrategy);
```

Comment:
>FilteringStrategy filteringStrategy [](start = 12, length = 35)

why did we decide to remove the support for user passing these values? My understanding for the design was: that the user would control the strategy and then splitter logic "splits" based on the user input.

Reply:
Our design has a built-in default: we filter by 'new' and 'updated' and we split by 'per run per target per rule'. Our emerging pipeline will use these defaults. I've eliminated the arguments and some supporting files as many of them are obsoleted by the current design. my thought is to get the defaults working and then to figure out how to make them generally configurable. this work technique is designed to avoid YAGNI and other costs related to reworking things that aren't immediately required.

---
In reply to: [308471467](https://github.com/microsoft/sarif-sdk/pull/1596#discussion_r308471467) [](ancestors = 308471467)

Reply:
ok

---
In reply to: [308472672](https://github.com/microsoft/sarif-sdk/pull/1596#discussion_r308472672) [](ancestors = 308472672,308471467)